ballerina/Ballerina.toml

```diff
@@ -6,7 +6,13 @@ license = ["Apache-2.0"]
 name = "ai.model.provider.anthropic"
 org = "ballerinax"
 repository = "https://github.com/ballerina-platform/module-ballerinax-ai.model.provider.anthropic"
-version = "1.0.0"
+version = "1.0.1"
 
 [platform.java21]
 graalvmCompatible = true
+
+[[platform.java21.dependency]]
+groupId = "io.ballerina.lib"
+artifactId = "ai.model.provider.anthropic-native"
```

Comment:
shall we change the artifact id to `ai..anthropic-native` ?

Reply:
Will change after pulling the RAG changes